release-note-script/src/main/java/ReleaseNoteCreation.java

```diff
@@ -301,7 +301,7 @@ private List<String> getPullRequestNumbers(String projectId) throws Exception {
       BufferedReader br =
           runSubProcessAndGetOutputAsReader(
               format(
-                  "gh project item-list %s --owner %s --limit %d | awk -F'\\t' '/%s\\t/ {print"
+                  "gh project item-list %s --owner %s --limit %d | awk -F'\\t' '/\\/%s\\t/ {print"
```

Comment:
Added a slash before the repository name. This is necessary to distinguish between scalardb and docs-internal-scalardb.